controller/app.controller.js

```diff
@@ -45,4 +45,18 @@ exports.getArticles = (req,res,next) => {
  
     res.status(200).send({articles:allArticlesArray})
   }).catch(next)
+}
+
+
+
+exports.getCommentsByArticleId = (req,res,next) => {
+  const {article_id} = req.params
+
```

Comment:
Not mandatory but I'd recommend getting a code formatter like Prettier, to improve this white space everywhere and make your code a bit easier to read 👓